Dockerfile

```diff
@@ -1,5 +1,5 @@
 ### Build web files
-FROM node:21-alpine AS web-build
+FROM node:22-alpine AS web-build
```

Comment:
If we're upgrading the `Node` version of the images, let's go to latest (`node:24-alpine`)

Reply:
Done